src/app/new-page/component/card/card.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+  attr.id="card-{{ cardIndex }}"
+  class="cardContent"
+  *ngIf="ready"
+  [hidden]="card.isHidden"
+  [ngClass]="{
+    hasHeading: !(card.listeners.length && card.dismissListeners.length)
```

Comment:
hmm, this seems weird that there is a `hasHeading` class that is dependent on if `listeners`/`dismissListeners` are defined.